examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -103,7 +118,43 @@ async function example_API_Delete(cacheClient: CacheClient) {
   }
 }
 
+function example_API_InstantiateAuthClient() {
+  new AuthClient({
+    credentialProvider: CredentialProvider.fromEnvironmentVariable({
+      environmentVariableName: 'MOMENTO_AUTH_TOKEN',
+    }),
+  });
+}
+
+async function example_API_GenerateAuthToken(authClient: AuthClient) {
+  const generateTokenResponse = await authClient.generateAuthToken(AllDataReadWrite, ExpiresIn.minutes(30));
+  if (generateTokenResponse instanceof GenerateAuthToken.Success) {
+    console.log('Auth token generated!');
```

Comment:
I wonder if it would be worthwhile to make it more obvious that this is an `AllDataReadWrite` token? Maybe like `AllDataReadWrite auth token generated` or `Auth token generated with AllDataReadWrite scope`?

Reply:
https://github.com/momentohq/client-sdk-javascript/pull/542/commits/dc0615785b66c8d18d2e4d3e217d8c42efb71353